src/gridDynMain/CMakeLists.txt

```diff
@@ -28,10 +22,18 @@ if(DIME_ENABLE)
 endif(DIME_ENABLE)
 
 foreach(keyfile IN LISTS KEY_LIBRARY_FILES)
-add_custom_command(TARGET gridDynMain POST_BUILD        # Adds a post-build event to gridDynMain
-    COMMAND ${CMAKE_COMMAND} -E copy_if_different  # which executes "cmake - E copy_if_different..."
-        "${keyfile}"      # <--this is in-file
-        $<TARGET_FILE_DIR:gridDynMain>)                 # <--this is out-file path
+    add_custom_command(TARGET
+                       gridDynMain
+                       POST_BUILD # Adds a post-build event to gridDynMain
+                       COMMAND ${CMAKE_COMMAND}
+                               -E
+                               copy_if_different # which executes "cmake - E
+                                                 # copy_if_different..."
+                               "${keyfile}" # <--this is in-file
+                               $<TARGET_FILE_DIR:gridDynMain>) # <--this is out-
+                                                               # file path
 endforeach(keyfile)
 
-INSTALL(TARGETS gridDynMain RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT runtime)
+install(TARGETS gridDynMain
+        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT
+        RUNTIME)
```

Comment:
runtime should not be capitalized,  should rename the component I guess

Reply:
This is a bug in cmake-format: https://github.com/cheshirekow/cmake_format/issues/66. We can rename it if that's what you want to do.

Reply:
Lets go ahead and rename it maybe "binaries"